src/client/containers/AddPeopleToRoom/AddPeopleToRoom.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import AddPeopleToRoomForm from '../../components/AddPeopleToRoom/AddPeopleToRoomForm';
+import '../../components/AddPeopleToRoom/AddPeopleToRoom.css';
+import AddNewRoom from '../AddNewRoom/AddNewRoom';
+
+export default function AddPeopleToRoom() {
+  const [users, setUsers] = useState([]); // this is use to control the data from api
+  const [addUsers, setAddUsers] = useState([]); //  this is use to add user to usersgroup
+  const [input, setInput] = useState([]); //  this is use to handle the input from user to search out
+  const [toAddRoom, setToAddRoom] = useState('false'); //  thi is use to move to next page
+  const url = 'api/users';
+  useEffect(() => {
+    fetch(url)
+      .then((res) => res.json())
+      .then((user) => setUsers(user));
+  }, []);
+
+  // Add users in group list by using click
+  const addUserInGroup = (e, id) => {
+    users.forEach((user) => {
```

Comment:
This is a bit too complicated, it can be simplified. But if you don't have time, ignore this comment.